src/lib/conn-manager-store.tsx

```diff
@@ -20,6 +20,9 @@ export interface ConnectionStoreItem {
   id: string;
   name: string;
   type: string;
+  createdAt?: string;
+  updatedAt?: string;
```

Comment:
Why string? We can use timestamp and currently the updatedAt and lastConnectedAt seem to be quite the same. When you connect, the updatedAt and lastConnectedAt update at the same time.